heron/api/src/java/com/twitter/heron/api/Config.java

```diff
@@ -229,6 +231,38 @@
   public static final String TOPOLOGY_UPDATE_REACTIVATE_WAIT_SECS =
       "topology.update.reactivate.wait.secs";
 
+  /**
+   * Packing algorithm used to calculate packing plan
+   */
+  public static final String TOPOLOGY_PACKING_ALGORITHM =
```

Comment:
I could see having multiple packing implementation supported, where power users could specify a non-default based on their needs. Or even write one of their own.